### PR TITLE
#185043752 Add Redis-mock

### PR DESCRIPTION
**Pull request by Edcreation**
#### What does this PR do?
- Add redis-mock in globals to mock redis client
#### Description of Task to be completed?
- Add redis mock 
#### How should this be manually tested?
- On running tests sinon stubs redisClient with redis-mock client.
#### Any background context you want to provide?
- Fixed tests delaying bug
#### What are the relevant pivotal tracker/Trello stories?
-- #185043752

### CODE REVIEW

**Review by AdrineUWERA — APPROVED** [review]

**Review by JosephNgabo — APPROVED** [review]

**Review by ishimwe5555 — APPROVED** [review]

**Review by princoo — APPROVED** [review]

File: src/database/config/config.js [hunk 1]
```diff
@@ -13,10 +13,7 @@ module.exports = {
     },
   },
   test: {
-    url: TEST_DATABASE_URL,
-    dialectOptions: {
-      ssl,
-    },
+    url: `postgresql://postgres:${process.env.POSTGRES_PASSWORD}@localhost:5432/postgres`,
```
Comment (munezerobagira) [inline src/database/config/config.js:16]:
```suggestion
    url: TEST_DATABASE_URL`,
```
can put the url in the environment variable instead

File: src/database/migrations/20230401104008-create-products.js [hunk 2]
```diff
@@ -37,10 +37,10 @@ module.exports = {
       collectionId: {
         type: Sequelize.UUID,
         allowNull: false,
+        onDelete: 'CASCADE',
         references: {
           model: 'collections',
           key: 'id',
-          onDelete: 'CASCADE',
         },
```
Comment (munezerobagira) [inline src/database/migrations/20230401104008-create-products.js:44]: Please don't edit migration, if you want to alter the table you can create new migration

File: src/helpers/redis.js [hunk 3]
```diff
@@ -1,10 +1,15 @@
 import { createClient } from 'redis';
+import dotenv from 'dotenv';
+
+dotenv.config();
```
Comment (munezerobagira) [inline src/helpers/redis.js:4]:
```suggestion
```

**Review by munezerobagira — CHANGES_REQUESTED** [review]

File: src/database/migrations/20230401104008-create-products.js (hunk 2 above)
Comment (Edcreation) [reply src/database/migrations/20230401104008-create-products.js]: Yes, Created another Migration instead.

File: src/database/config/config.js (hunk 1 above)
Comment (Edcreation) [reply src/database/config/config.js]: Resolved.

**Review by mirindi19 — APPROVED** [review]

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**